projects/packages/jetpack-mu-wpcom/eslint.config.mjs

```diff
@@ -7,9 +7,6 @@ export default [
 		rules: {
 			'testing-library/prefer-screen-queries': 'off',
 			'react/jsx-no-bind': 'off',
-			// @todo Turn these back on for JS here. The block below handles TS.
-			'jsdoc/require-returns': 'off',
```

Comment:
Looks like I had a typo here, that should have been `jsdoc/require-returns-type` like below and in [the old code](https://github.com/Automattic/jetpack/blob/44813737107aa90c148fc9a6a185d12e02b6d06d/projects/packages/jetpack-mu-wpcom/.eslintrc.js#L15). 😳